utPlayer.py

```diff
@@ -75,8 +97,12 @@ def stop(self):
         Args:
             None
         """
-        if (self.playbackTool == "gstreamer"):
-            self.session.write("\x03")  # CNTRL-C
+        cmd = self.player.get("stop_command")
```

Comment:
you will get an exception if stop_command isn't present. It's a fatal.

`self.session.write( self.player.stop_command )`